esx_service/cli/vmdkops_admin.py

```diff
@@ -857,8 +859,8 @@ def tenant_rm(args):
     # If args "remove_volumes" is not specified in CLI
     # args.remove_volumes will be None
     if args.remove_volumes:
-        if args.remove_volumes == 'True':
-            remove_volumes = True
+        print "All Volumes will be removed"
```

Comment:
Done. Looks like all "print" in this file are incompatible with Python 3. I don't want to pollute change so only changed newly added print @ L862.